includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -336,6 +336,14 @@ class AMP_Style_Sanitizer extends AMP_Base_Sanitizer {
 	 */
 	private $remote_request;
 
+	/**
+	 * Cached call to is_customize_preview()
+	 *
+	 * @see is_customize_preview()
+	 * @var bool
+	 */
+	private $is_customize_preview;
```

Comment:
Why would the call to `is_customize_preview()` need to be cached? At a quick glance the function does not seem to be performance intensive in any way.

Reply:
You're right, but the function will be called _many_ times and I wanted to eek our every possible performance gain.

Reply:
In other words, by caching the output of the function call, there we can save two function invocations multiplied by every selector in the stylesheets.

Reply:
Yea that makes sense. A PR check to measure the performance delta between changes would be awesome right now :smile:.